src/components/network-chart/index.css

```diff
@@ -31,7 +31,8 @@
   background-color: #f5f5f5;
 }
 
-.networkChartRowItem:hover {
+.networkChartRowItem:hover,
+.networkChartRowItem.isRightClicked {
   background-color: #bbe0f6;
```

Comment:
I didn't want to change that here but we should make the color closer to what we have in other panels.